.ci/pipeline_definitions

```diff
@@ -33,6 +33,11 @@ machine-controller-manager-provider-aws:
     pull-request:
       traits:
         pull-request: ~
+    create-upgrade-prs:
+      traits:
+        cronjob:
+          interval: '5m'
+        update_component_deps: ~
```

Comment:
This code snippet will add a new pipeline job which will poll for new versions of dependent components. Since the  machine controller manager is the only dependency only its version will be checked